ext/ddtrace.stub.php

```diff
@@ -0,0 +1,496 @@
+<?php
+
+/** @generate-class-entries */
+
+// phpcs:disable PSR1.Classes.ClassDeclaration.MultipleClasses
+
+namespace DDTrace {
+
+    class SpanData {
+        /**
+         * @var string The span name (Optional as of ddtrace v0.47.0; defaults to ‘ClassName.methodName’ if not set)
+         */
+        public string $name;
+
+        /**
+         * @var string The resource you are tracing (Optional as of ddtrace v0.47.0; defaults to SpanData::$name
+         * if not set)
+         */
+        public string $resource;
+
+        /**
+         * @var string The service you are tracing
```

Comment:
Defaults to active service at the time of span creation (i.e. parent span), or datadog.service ini setting if no parent exists.